portability-transfer/src/main/java/org/datatransferproject/transfer/copier/PortabilityAbstractInMemoryDataCopier.java

```diff
@@ -126,7 +126,7 @@ protected ExportResult<?> copyIteration(
     if (exportedData != null) {
       PortabilityJob job = jobStore.findJob(jobId);
       TransferMode transferMode =
-          job.transferMode() == null ? TransferMode.DATA_TRANSFER : job.transferMode();
+          job == null ? TransferMode.DATA_TRANSFER : job.transferMode();
```

Comment:
looks like we already use Preconditions.checkNotNull library in DTP; maybe just fail on the first line of this entire function (copyIteration with a valid job id is a larger/different issue), and then leave this code as is?


see [`Preconditions.checkNotNull(id, "transfer job ID required for GetTransferJobAction");`](https://github.com/google/data-transfer-project/blob/a81eaed0ba1ecbbdd14ff4a4e844a27a09fdd537/portability-api/src/main/java/org/datatransferproject/api/action/transfer/GetTransferJobAction.java#L37)

Reply:
Changed the code as suggested